lib/deprecated.js

```diff
@@ -28,13 +28,13 @@ exports.defaultMsg = function(packageName, funcName) {
 };
 
 exports.printWarning = function(msg) {
-    // Watch out for IE7 and below! :(
     /* istanbul ignore next */
-    if (typeof console !== "undefined") {
-        if (console.info) {
-            console.info(msg);
-        } else {
-            console.log(msg);
-        }
+    if (typeof process === "object" && process.emitWarning) {
```

Comment:
Can we just put this above the code that was there before? The `return` statements returning `undefined` don't seem very useful.

Reply:
@fearphage I think that's what I did. The original code is still there, I just removed the `else`, and the check for `console`, does that make sense?

Reply:
@mshaaban0 I can see the original code is removed based on the diff. You replaced the `if-else` statements with `return`ing `undefined` instead. Returning `console.*` statements holds no value.

I think the only line that needs to change should be the [old (removed) line 33](https://github.com/sinonjs/commons/pull/37/files#diff-83f8fa045602d9dd487b5b2b8e1e8a89L33) could be an `else if` after your new code. This should be a mostly additive change instead of a style change as well.